cloudinit/userdata_cloudinit.erb

```diff
@@ -41,6 +49,28 @@ users:
    lock-passwd: false
    passwd: <%= @host.root_pass %>
 
+ runcmd:
+ <% if rhel_compatible -%>
+ - <%= snippet('epel') -%>
```

Comment:
Sorry, it wasn't really the indenting of the array I meant, it was more that I expected the template to render more than one line. I think that's fine then, provided nobody edits it!